<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemDialog</class>
 <widget class="QWidget" name="AddItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Items to %1</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="SelectItem" name="w_select_item" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SelectColor" name="w_select_color" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>10</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="PictureWidget" name="w_picture">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PriceGuideWidget" name="w_price_guide">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="AppearsInWidget" name="w_appears_in" native="true"/>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;Quantity</string>
           </property>
           <property name="buddy">
            <cstring>w_qty</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="w_qty">
           <property name="maximum">
            <number>2000000000</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="w_bulk_label">
           <property name="text">
            <string>&amp;Bulk</string>
           </property>
           <property name="buddy">
            <cstring>w_bulk</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="w_bulk">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>2000000000</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="w_label_currency">
           <property name="text">
            <string>&amp;Price (%1)</string>
           </property>
           <property name="buddy">
            <cstring>w_price</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="w_price"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Total</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="w_total">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="w_tier_label">
           <property name="text">
            <string>&amp;Tiered prices</string>
           </property>
           <property name="buddy">
            <cstring>w_radio_percent</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" rowspan="2" colspan="3">
          <layout class="QGridLayout">
           <item row="0" column="0">
            <widget class="QSpinBox" name="w_tier_qty_0"/>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="w_tier_qty_1"/>
           </item>
           <item row="0" column="2">
            <widget class="QSpinBox" name="w_tier_qty_2"/>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="w_tier_price_0"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="w_tier_price_1"/>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="w_tier_price_2"/>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout">
           <item>
            <widget class="QRadioButton" name="w_radio_percent">
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="w_radio_currency">
             <property name="text">
              <string notr="true">%1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Condition</string>
           </property>
           <property name="buddy">
            <cstring>w_radio_new</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="3">
          <layout class="QHBoxLayout">
           <item>
            <widget class="QRadioButton" name="w_radio_new">
             <property name="text">
              <string>&amp;New</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="w_radio_used">
             <property name="text">
              <string>&amp;Used</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>271</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="w_comments_label">
           <property name="text">
            <string>Co&amp;mments</string>
           </property>
           <property name="buddy">
            <cstring>w_comments</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="QLineEdit" name="w_comments"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="w_remarks_label">
           <property name="text">
            <string>&amp;Remarks</string>
           </property>
           <property name="buddy">
            <cstring>w_remarks</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="3">
          <widget class="QLineEdit" name="w_remarks"/>
         </item>
         <item row="7" column="1">
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QCheckBox" name="w_toggle_picture">
       <property name="icon">
        <iconset resource="brickstore.qrc">
         <normaloff>:/images/edit_bl_catalog.png</normaloff>:/images/edit_bl_catalog.png</iconset>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="w_toggle_price_guide">
       <property name="icon">
        <iconset resource="brickstore.qrc">
         <normaloff>:/images/edit_bl_priceguide.png</normaloff>:/images/edit_bl_priceguide.png</iconset>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="w_toggle_appears_in">
       <property name="icon">
        <iconset resource="brickstore.qrc">
         <normaloff>:/images/appearsin.png</normaloff>:/images/appearsin.png</iconset>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10000</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="w_merge">
       <property name="text">
        <string>&amp;Consolidate with list</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>11</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="w_buttons">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectItem</class>
   <extends>QWidget</extends>
   <header>selectitem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PriceGuideWidget</class>
   <extends>QFrame</extends>
   <header>priceguidewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PictureWidget</class>
   <extends>QFrame</extends>
   <header>picturewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AppearsInWidget</class>
   <extends>QWidget</extends>
   <header>appearsinwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectColor</class>
   <extends>QWidget</extends>
   <header>selectcolor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="brickstore.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>w_toggle_appears_in</sender>
   <signal>toggled(bool)</signal>
   <receiver>w_appears_in</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w_toggle_picture</sender>
   <signal>toggled(bool)</signal>
   <receiver>w_picture</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w_toggle_price_guide</sender>
   <signal>toggled(bool)</signal>
   <receiver>w_price_guide</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w_buttons</sender>
   <signal>rejected()</signal>
   <receiver>AddItemDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
